$OpenBSD: patch-test_test_cpp,v 1.2 2015/10/20 14:08:57 sthen Exp $

Revert post-2.2.1 commit which causes hangs at startup on OpenBSD
https://github.com/Icinga/icinga2/commit/86819f446e20f6aa0efab45d1e9e7eb87a2bd2b1.patch

https://dev.icinga.org/issues/10410

--- test/test.cpp.orig	Mon Oct 19 10:14:40 2015
+++ test/test.cpp	Tue Oct 20 14:19:55 2015
@@ -31,11 +31,12 @@ struct InitLibBase
 	InitLibBase(void)
 	{
 		Application::InitializeBase();
+		Timer::Initialize();
 	}
 
 	~InitLibBase(void)
 	{
-		Application::UninitializeBase();
+		Timer::Uninitialize();
 	}
 };
 
